containerscan/rawdatastrucutres.go

```diff
@@ -39,6 +39,15 @@ type ScanResultReportV1 struct {
 	PaginationInfo  apis.PaginationMarks                 `json:"paginationInfo"`
 }
 
+type ScanResultReportV2 struct {
```

Comment:
[TODO]: I think we should add a field indicating whether we ran relevancy or not on the report.
Meaning, we should have a boolean (not a pointer) indicating this.
This will enable the BE to determine right away if the report contains any relevant data or not.